src/custom-elements/src/utils/Drag.ts

```diff
@@ -0,0 +1,87 @@
+let self: Drag;
+
+export class Drag {
+  private trigger: HTMLElement;
+  private elementToMove: HTMLElement;
+  private initialTransformStyling: string;
+  private pos1: number;
+  private pos2: number;
+  private pos3: number;
+  private pos4: number;
```

Comment:
What about using vectors instead of separate numbers? For example: startPosition and endPosition of type Vector